tests/src/test/scala/whisk/core/containerpool/kubernetes/test/KubernetesClientTests.scala

```diff
@@ -188,6 +189,7 @@ object KubernetesClientTests {
   implicit def strToInstant(str: String): Instant =
     strToDate(str).get
 
+  implicit val as = ActorSystem("kubernetes-client-tests-actor-system")
```

Comment:
This actorSystem is leaked I think. Could you make the `TestKubernetesClient` take the actorSystem as an implicit parameter, so you can use the one imported (and closed) by the tests above?

Or maybe even move the class into the testclass. Makes it even easier?

Reply:
I will give it a try, but it isn't clear why this test was setup this way? @dgrove-oss @jcrossley3 ?

Reply:
The `TestKubernetesClient` is shared with `KubernetesContainerTests` - so for now, changed to implicit ActorSystem (and left object/classes in current places). Good?

Reply:
I don't remember any particular reason;  I think it may be as simple as that since we hadn't needed to have our hands on an ActorSystem before in the stubbed out test client it wasn't plumbed through.